backend/CircleForms/Models/Posts/PostFields/PostField.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using CircleForms.Models.Enums;
 
-namespace CircleForms.Models.PostFields;
+namespace CircleForms.Models.Posts.PostFields;
 
 public class PostField
 {
     protected PostField() { }
     public virtual FieldType Type { get; set; }
 
-    public string Title { get; set; }
+    public string Value { get; set; }
```

Comment:
Why `value`? It was supposed to be a title of a field, for example "how much pp was cookiezi's blue zenith score worth?"